modules/serverless-config-replicas-eventing.adoc

```diff
@@ -0,0 +1,49 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-ha.adoc
+
+[id="serverless-config-replicas-eventing_{context}"]
+= Configuring high availability replicas for Eventing
+
+.Prerequisites
+* An {product-title} account with cluster administrator access.
+* Installed the {ServerlessOperatorName} and Knative Eventing.
+
+.Procedure
+
+. In the {product-title} web console *Administrator* perspective, navigate to *OperatorHub* -> *Installed Operators*.
++
+image::serving-installed-operator.png[Installed Operators page]
++
+. Select the `knative-eventing` namespace.
++
+. Click *Knative Eventing* in the list of *Provided APIs* for the {ServerlessOperatorName} to go to the *Knative Eventing* tab.
++
+image::eventing-tab-created.png[Knative Eventing tab]
++
+. Click *knative-eventing*, then go to the *YAML* tab in the *knative-eventing* page.
++
+image::eventing-YAML-HA.png[Knative Eventing YAML]
++
+. Edit the custom resource definition YAML:
++
+.Example YAML
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  high-availability:
+    replicas: 3
+----
++
+[IMPORTANT]
+====
+Do not modify any YAML contained inside the `config` field. Some of the configuration values in this field are injected by the {ServerlessOperatorName}, and modifying them will cause your deployment to become unsupported.
+====
++
+* The default `replicas` value is `2`.
```

Comment:
For Eventing, the default ist currently set to `1`:

There is work in place to set it to two: 
https://github.com/openshift-knative/serverless-operator/pull/919